onadata/apps/main/views.py

```diff
@@ -1086,7 +1091,11 @@ def form_photos(request, username, id_string):
     image_urls = json.dumps(image_urls)
 
     data['images'] = image_urls
-    data['profile'], created = UserProfile.objects.get_or_create(user=owner)
+    data['profile'] = property(
+        lambda u: cache_single_object(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected keyword argument 'id' in function call](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5478081)